src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -245,7 +247,7 @@ private Container createCompoundContainer(PopulationSimulation populationSimulat
          var compoundContainer = new Container().WithName(compoundName);
          populationSimulation.PKAnalyses.AllPKParametersFor(compoundName).Each(quantityPKParameter =>
          {
-            double? defaultValue = quantityPKParameter.ValuesAsArray.Median();
```

Comment:
That was most of the bug. This extension method expected the array to be sorted. But when you write the code like that it looks like it should work.

I added this more specific object to use for quantile type calculations on float arrays just to make it more obvious what the requirements are